neurom/fst/_core.py

```diff
@@ -129,12 +131,28 @@ def _check_soma_topology_swc(points):
     'SWC': _remove_soma_initial_point,
     'H5V1': None,
     'H5V2': None,
-    'NL-ASCII': None
+    'NL-ASCII': None,
 }
 
-_SOMA_ACTION = {
-    'SWC': _check_soma_topology_swc,
-    'H5V1': None,
-    'H5V2': None,
-    'NL-ASCII': None
+_SOMA_CONFIG = {
+    # after much debate (https://github.com/BlueBrain/NeuroM/issues/597)
```

Comment:
I would say that a link to the issue is enough if these sources are already mentioned there